### ISSUE TITLE
User Guide: Keyboard Shortcuts

### ISSUE THREAD

**Issue by RyanDuff613** [SketchIO/feedback-summer-2020#27]
Good chance you're already aware of this but just noticing some discrepancies between the keyboard shortcuts listed in the user guide. 

| keyboard input      | user guide         | my result  |
| :--- |:---| :---|
|1|Select|Vector Fill|
|2|Text|Select|
|3|Pen|Crop and Resize|
|4|Arrow|Pen|
|5|Path|Caligraphy|
|6|Stamp|Tile Brush|
|7|Star|Mirror Brush|
|8|Clipart|Star|
|9|Floodfill|Clipart|

**Comment by ryanreece** [comment]
Potentially have a button on the UI which will re-bind the keyboard shortcuts...

**Comment by mudcube** [comment]
I dig it, lets create an issue for UIConfig to re-bind shortcuts, but in the meantime, let's figure out the best presets to go with, and synchronize the app with the guide.  Following the interface, maybe:

1. Select
2. Crop
3. Tile Brush
4. Calligraphy
5. Mirror Brush
6. Star
7. Clipart
8. Text
9. Vector Fill

What do you think?

As far as the UIConfig interface, I'm thinking there would be 9 squares, which would initially have the preset tool icons for the hotKeys.  All in a single row, as to not take up much space.  When you click on the squares, you'd get a UIPopoutGrid of tools you can choose for that hotKey.  It would take about an hour or so to implement.

**Comment by mudcube** [comment]
Added a HotKeys configurator to the Settings Pane :)  Will be available upon next release.

**Comment by mudcube** [comment]
<img width="410" alt="Screen Shot 2020-09-05 at 12 51 19 AM" src="https://user-images.githubusercontent.com/101564/92300723-ef825500-ef11-11ea-9308-0f63ef774144.png">

**Comment by mudcube** [comment]
@ryanreece Ping.

**Comment by ryanreece** [comment]
@mudcube Yep, this looks good to me! Thanks for this nice customization :)

**Comment by mudcube** [comment]
@ryanreece Don't forget to add to the guide thooo!

**Comment by ryanreece** [comment]
Whoops, that's right! Forgot to read the whole thing. Moving too quickly.... :sigh:
